src/App.js

```diff
@@ -0,0 +1,28 @@
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Routes,
+  Link,
+} from "react-router-dom";
+import Login from "./auth/login";
+import Register from "./auth/register";
+import Landing from "./landing";
+import Messenger from "./messenger";
+
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Routes>
+          <Route path="login" element={<Login />} />
+          <Route path="register" element={<Register />} />
+          <Route path="messenger" element={<Messenger />} />
+          <Route path="" element={<Landing />} />
```

Comment:
Can you also create a 404 page @Ajy440 ?
So if url doesn't exist it will go to that page